src/tokens/Assets.test.tsx

```diff
@@ -336,6 +336,21 @@ describe('AssetsScreen', () => {
     expect(navigate).toHaveBeenCalledWith(Screens.DappShortcutsRewards)
   })
 
+  it('clicking Import Token opens a screen', () => {
+    jest.mocked(getFeatureGate).mockReturnValue(true)
```

Comment:
It could make sense to test the reverse state as well (when the feature flag is not enabled).

Reply:
agree with this - there is more than 1 feature gate used in this Assets component so we should really be mocking when `getFeatureGate(StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW)` returns true and false